kingnodes/chains.json

```
@@ -1,78 +1,87 @@
-      "address": "chihuahuavaloper10wxn2lv29yqnw2uf4jf439kwy5ef00qdqj3xsc",
-      "restake": {
-        "address": "chihuahua1xs023g3yj2mavwn3cjclgsh8mlk0kgsj9trcv8",
-        "run_time": "2x per day",
-        "minimum_reward": 100000
+   "$schema":"../chains.schema.json",
+   "name":"kingnodes",
+   "chains":[
+      {
+         "name":"chihuahua",
+         "address":"chihuahuavaloper10wxn2lv29yqnw2uf4jf439kwy5ef00qdqj3xsc",
+         "restake":{
+            "address":"chihuahua1xs023g3yj2mavwn3cjclgsh8mlk0kgsj9trcv8",
+            "run_time":"2x per day",
+            "minimum_reward":100000
+         }
+      },
+      {
+         "name":"evmos",
+         "address":"evmosvaloper1x5y65hgngh77sxta6s2vpjnfjep3ltllyhf2hs",
+         "restake":{
+            "address":"evmos16e67hq4eqgnfw6pxu8v33mz259z5r30d9v2uq2",
+            "run_time":"2x per day",
+            "minimum_reward":100000000000000000
+         }
+      },
+      {
+         "name":"juno",
+         "address":"junovaloper10wxn2lv29yqnw2uf4jf439kwy5ef00qdelfp7r",
+         "restake":{
+            "address":"juno1xs023g3yj2mavwn3cjclgsh8mlk0kgsjsvdd2e",
+            "run_time":"2x per day",
+            "minimum_reward":100000
+         }
+      },
+      {
+         "name":"stargaze",
+         "address":"starsvaloper10wxn2lv29yqnw2uf4jf439kwy5ef00qd0jluj0",
+         "restake":{
+            "address":"stars1xs023g3yj2mavwn3cjclgsh8mlk0kgsjjzetx5",
+            "run_time":"2x per day",
+            "minimum_reward":100000
+         }
+      },
+      {
+         "name":"firmachain",
+         "address":"firmavaloper192xjmnzsdf56nhra452p67ed7uw2m7qyrz0kgu",
+         "restake":{
+            "address":"firma1e25tjsmzzex9dcs5qzn7v0e65ra8kqn54lpsak",
+            "run_time":"2x per day",
+            "minimum_reward":100000
+         }
+      },
+      {
+         "name":"kichain",
+         "address":"kivaloper10wxn2lv29yqnw2uf4jf439kwy5ef00qd46r89v",
+         "restake":{
+            "address":"ki1xs023g3yj2mavwn3cjclgsh8mlk0kgsjhnlef3",
+            "run_time":"2x per day",
+            "minimum_reward":100000
+         }
+      },
+      {
+         "name":"kujira",
+         "address":"kujiravaloper10wxn2lv29yqnw2uf4jf439kwy5ef00qdxdd7nr",
+         "restake":{
+            "address":"kujira1xs023g3yj2mavwn3cjclgsh8mlk0kgsjhkvwq0",
+            "run_time":"2x per day",
+            "minimum_reward":100000
+         }
+      },
+      {
+         "name":"mars",
+         "address":"marsvaloper10wxn2lv29yqnw2uf4jf439kwy5ef00qdepeenc",
+         "restake":{
+            "address":"mars1xs023g3yj2mavwn3cjclgsh8mlk0kgsjmrh0c7",
+            "run_time":"2x per day",
+            "minimum_reward":100000
+         }
+      },
+      {
+         "name":"nois",
+         "address":"noisvaloper10wxn2lv29yqnw2uf4jf439kwy5ef00qdqcn7h2",
+         "restake":{
+            "address":"nois1xs023g3yj2mavwn3cjclgsh8mlk0kgsjhknupa",
+            "run_time":"2x per day",
```

`2x per day` like this will cause problems. It needs to be an array of specific times in UTC, i.e.

```suggestion
            "run_time":["09:00", "21:00"],
```

Alternatively could be `every 12 hours` but better to be specific with what time you'll run it.